homeassistant/helpers/schema_config_entry_flow.py

```diff
@@ -188,7 +191,10 @@ def _show_next_step(
 
         # Show form for next step
         return self._handler.async_show_form(
-            step_id=next_step_id, data_schema=data_schema, errors=errors
+            step_id=next_step_id,
+            data_schema=data_schema,
+            errors=errors,
+            last_step=True if form_step.next_step is None else None,
```

Comment:
This is not aligned with logic in frontend: https://github.com/home-assistant/frontend/blob/e7af27b400130117eea7f4f547656160b8a3c489/src/dialogs/config-flow/step-flow-form.ts#L72-L76

Instead, try:
```suggestion
            last_step=form_step.next_step is None,
```